python/lsst/ts/mtdome/mock_llc/mock_motion/elevation_motion.py

```diff
@@ -197,21 +182,5 @@ def go_stationary(self, start_tai: float) -> float:
         self._commanded_motion_state = LlcMotionState.STATIONARY
         return 0.0
 
-    def exit_fault(self, start_tai: float) -> None:
-        """Clear the fault state.
-
-        Parameters
-        ----------
-        start_tai: `float`
-            The TAI time, unix seconds, at which the command was issued. To
-            model the real dome, this should be the current time. However, for
-            unit tests it can be convenient to use other values.
-        """
-        position, velocity, motion_state = self.get_position_velocity_and_motion_state(
-            tai=start_tai
-        )
-        self._start_tai = start_tai
-        self._start_position = position
-        self._end_position = position
-        self._crawl_velocity = 0.0
-        self._commanded_motion_state = LlcMotionState.STATIONARY
+    def set_fault(self, start_tai: float, drives_in_error: list[int]) -> None:
+        raise NotImplementedError
```

Comment:
Please document why this is not supported.

Reply:
Done.